pkg/capabilities/capabilities.go

```diff
@@ -0,0 +1,25 @@
+package capabilities
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/syndtr/gocapability/capability"
+)
+
+func CheckRequiredCapabilities(caps capability.Capabilities, reqCaps []capability.Cap) error {
```

Comment:
As commented on @michaelkatch PR, the capabilities packages has enough meaning to say that we're checking required capabilities.

```suggestion
func CheckRequired(caps capability.Capabilities, reqCaps []capability.Cap) error {
```